test/algorithms/test_warm_start_qaoa.py

```diff
@@ -52,13 +57,19 @@ def test_max_cut(self):
         backend = BasicAer.get_backend("statevector_simulator")
         qaoa = QAOA(quantum_instance=backend, reps=1)
         aggregator = MeanAggregator()
+        # a list of converters without max2min
+        ineq2eq = InequalityToEquality()
+        int2bin = IntegerToBinary()
+        penalize = LinearEqualityToPenalty()
+        converters = [ineq2eq, int2bin, penalize]
         optimizer = WarmStartQAOAOptimizer(
             pre_solver=presolver,
             relax_for_pre_solver=False,
             qaoa=qaoa,
             epsilon=0.25,
             num_initial_solutions=10,
             aggregator=aggregator,
+            converters=converters,
```

Comment:
Because maxcut does not require any converters, the following should work.
```suggestion
            converters=[]
```